Training3/src/tasks/src/App.js

```diff
@@ -0,0 +1,30 @@
+import "./App.css";
+import { useEffect, useState } from "react";
+import Modal from "./Modal";
+
+function App() {
+  const [isOpen, setIsOpen] = useState(false);
+  function handleClick() {
+    setIsOpen(!isOpen);
+  }
+  useEffect(() => {
+    if (isOpen === true) {
+      const modal = document.querySelector(".Modal");
```

Comment:
@phuongluongZigvy we don't use the document.querySelector, please try to use React ref